packages/shaderlab/src/shaders/shadingPBR/LightDirectPBR.glsl

```diff
@@ -34,7 +37,9 @@ void surfaceShading(Varyings varyings, SurfaceData surfaceData, BRDFData brdfDat
     FUNCTION_DIFFUSE_LOBE(varyings, surfaceData, brdfData, attenuationIrradiance, diffuseColor);
     // Specular Lobe
     FUNCTION_SPECULAR_LOBE(varyings, surfaceData, brdfData, incidentDirection, attenuationIrradiance, specularColor);
-
+    // Iridescence Lobe
+    FUNCTION_IRIDESCENCE_LOBE(varyings, surfaceData, brdfData, incidentDirection, attenuationIrradiance, specularColor);
```

Comment:
`FUNCTION_IRIDESCENCE_LOBE` is mixed logic with `FUNCTION_SPECULAR_LOBE`, so it should be written directly in `FUNCTION_SPECULAR_LOBE`